tester/reporter_test.go

```diff
@@ -80,11 +95,16 @@ data.foo.bar.test_corge: FAIL (0s)
 
 SUMMARY
```

Comment:
Am I right that all of the below is printed with `--verbose` on? I don't see the `SUMMARY` headline without verbose mode... so it would be good to have something that tested output without it.

Reply:
Good point, updating the test for non verbose :+1: